src/Security/ContaoAuthenticator.php

```diff
@@ -28,6 +30,21 @@
 class ContaoAuthenticator implements SimplePreAuthenticatorInterface
 {
     /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * Constructor.
+     *
+     * @param ContainerInterface $container The service container
+     */
+    public function __construct(ContainerInterface $container)
```

Comment:
True. Though we could also say this is a bug (see my other comment) in the RC ;-)